headerItems.js

```diff
@@ -0,0 +1,32 @@
+const request = require('request');
+
+const fs = require('fs');
+
+let output = '';
+
+request('https://refdata-api.azurewebsites.net/api/fullheadermenu', (error, response, body) => {
```

Comment:
Can this URL be an ENV var please as it will allow it to be changed without a release.

Reply:
I've added the URL to vault `HEADER_API_URL` could you please add it where required in the code please?